Changelog.md

```diff
@@ -79,6 +79,7 @@ Bugfixes:
  * References Resolver: Enforce ``storage`` as data location for mappings.
  * References Resolver: Report error instead of assertion fail when FunctionType has an undeclared type as parameter.
  * Type Checker: Consider fixed size arrays when checking for recursive structs.
+ * Type Checker: Report error instead of crash when using structs in events without experimental ABIEncoderV2.
```

Comment:
Note that it did not always crash. In some cases it just logged the memory or storage pointer as a uint256.